.ci_support/linux_64_.yaml

```diff
@@ -6,3 +6,11 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.12.* *_cpython
+python_min:
+- '3.9'
```

Comment:
> You may need to override the python_min variable in the `conda_build_config.yaml`/`variants.yaml` if the package requires a newer Python version than the currently supported minimum version on conda-forge.

Okay, so I now need to figure out how to override `python_min` as to be '3.8'.

Reply:
@beckermr @chrisburr @jakirkham This isn't very clear from https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#general-pinning-examples (as there's nothing on this in https://conda-forge.org/docs/). Do we need to create a `recipe/conda_build_config.yaml` that is _just_

```yaml
python_min:
- '3.8'
```

or do we need to include all possible Python information in it, essentially duplicating all information in `.ci_support/linux_64_.yaml`

```yaml
pin_run_as_build:
  python:
    min_pin: x.x
    max_pin: x.x
python:
- 3.12.* *_cpython
python_min:
- '3.8'
```

?

Reply:
cc @dopplershift given his participation (and shared concerns on supporting older Pythons) in https://github.com/conda-forge/conda-forge.github.io/issues/2210. Ryan, maybe you have insight here as to how this works?

Reply:
Hopefully that works. If not, we can try other things.

Reply:
Thanks very much @beckermr. That indeed does work. :+1: 

If I find time this evening I will open and Issue and PR to make things a bit more explicit for people who have only been doing this for a few years like me.

Reply:
Improvements to the docs would certainly be welcome 🙂

Writing good docs seems to be one of the hardest problems in open source. So we welcome all the help we can get 😅

Reply:
It's hard, and also the conda-forge team has an nearly impossible task of covering a _huge_ landscape of complicated infrastructure (that still amazingly "just works"!).

Given that I'm trying to unbreak some Scikit-HEP related builts that might not happen tonight, but my goal is to open up a PR in the next 24 hours to iterate with you all on. :+1:

Reply:
Okay, so closer to 96 hours, but: https://github.com/conda-forge/conda-forge.github.io/pull/2370